### PR TITLE
JAPI-40 Generated and versioned

### PR DESCRIPTION
**Pull request by Michael-Gardner**
- Updated generated wsdl stub code
- Added wsesdlconfig
- Versioned to 5.6.0
- Latest = 6.0.0

Signed-off-by: Michael Gardner <michael.gardner@lexisnexis.com>

### CODE REVIEW

File: wsclient/.settings/org.eclipse.core.resources.prefs [hunk 1]
```diff
@@ -1,4 +1,7 @@
 eclipse.preferences.version=1
```
Comment (rpastrana) [inline wsclient/.settings/org.eclipse.core.resources.prefs]: add this file to the git ignore list

File: wsclient/.settings/org.eclipse.jdt.core.prefs [hunk 2]
```diff
@@ -1,11 +1,12 @@
 eclipse.preferences.version=1
```
Comment (rpastrana) [inline wsclient/.settings/org.eclipse.jdt.core.prefs]: git ignore

**Comment by Michael-Gardner** [comment]
@rpastrana 

- Cleaned up the .gitignore files.  Added in .settings/ directory and removed the wsclient/.settings/ files from the git cache so they should no longer show up when you do a commit or git status.
- Fixed test files for new versions of wsclient
- Changes to client files to fix WuActions -> ECLWUActions change
- Changes to client files for ApplicationValues instead of each individual name.  I kept the function signature the same though (it accepts appKey, appName, appData which get translated to application, name, value respectively.)  Thinking about adding a second signature that takes an ApplicationValues[] array as an argument.  Ideas?

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,10 @@
 */target
 bin/
+.project
+.settings/
+EclRecord.tokens
```
Comment (rpastrana) [inline .gitignore]: good idea

File: wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java [hunk 4]
```diff
@@ -700,9 +673,8 @@ public WUQueryResponse workUnitUQuery(String wuid, String jobname, String cluste
             if (jobname != null) request.setJobname(jobname);
             if (count != null) request.setCount(count);
             if (owner != null) request.setOwner(owner);
-            if (appKey != null) request.setApplicationKey(appKey);
-            if (appData != null) request.setApplicationData(appData);
-            if (appName != null) request.setApplicationName(appName);
+            ApplicationValue applicationValue[] = {new ApplicationValue(appKey,appName,appData)};
```
Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java]: do we have a version that accepts appvalues ?

Comment (Michael-Gardner) [inline wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java]: Not at the moment.  I was thinking of adding a signature in that would replace the appName, appKey, appData parameters with an ApplicationValue[] parameter.  I'd use the code from the current implementation and then just make the signature we have currently (for backwards compatibility) be a wrapper to the new sig.

Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java]: Let's add that new wrapper now.

**Comment by rpastrana** [comment]
looks reasonable.

**Pull request merged**